docs/.c4/model.c4

```diff
@@ -0,0 +1,596 @@
+
+          // external bi-directional relationships
+          pepr-admission -> pepr-mutating-webhook {
+            style {
+              head normal
+              tail normal
+            }
+          }
+
+          pepr-admission -> pepr-validating-webhook {
+            style {
+              head normal
+              tail normal
+            }
+          }
+        }
+      }
+
+      zarf = coreNamespace {
+        title 'Zarf Namespace'
+        summary ' '
+        description 'Namespace containing Zarf components for package deployment and image registry. Zarf enables air-gapped deployments by packaging and deploying UDS Core with all dependencies, including container images, Helm charts, and manifests.'
+
+        zarf-agent = pod {
+          title 'Zarf Agent'
+          summary ' '
+          description 'Mutates pod specifications to inject Zarf registry configuration. The agent intercepts pod creation requests and rewrites image references to point to the local Zarf registry, enabling air-gapped deployments without modifying application manifests.'
+
+          // external bi-directional relationship
+          zarf-agent -> zarf-mutating-webhook {
+            style {
+              head normal
+              tail normal
+            }
+          }
+        }
+
+        zarf-registry = pod {
+          title 'Zarf Registry'
+          summary ' '
+          description 'Local container image registry for air-gapped deployments. Hosts all container images required by UDS Core and deployed applications, eliminating the need for external registry access in disconnected environments.'
+
+          // external bi-directional relationship
+          zarf-registry -> kubernetes-control-plane.kubelet {
+            style {
+              head normal
+              tail normal
+            }
+          }
+        }
+      }
+
+      istio-system = coreNamespace {
+        title 'Istio System Namespace'
+        summary ' '
+        description 'Namespace containing core Istio service mesh control plane components. UDS Core uses Istio to provide mTLS encryption, traffic management, and zero-trust networking. Supports both sidecar injection and ambient mesh modes for flexible deployment options.'
+
+        istiod = pod {
+          title 'Istiod'
+          summary ' '
+          description 'Istio control plane managing service mesh configuration and certificate issuance. Istiod handles service discovery, certificate rotation for mTLS, configuration distribution to proxies, and manages both Envoy sidecars and ztunnel proxies in ambient mode. Integrates with Pepr for dynamic authorization policy generation.'
```

Comment:
```suggestion
          description 'Istio control plane managing service mesh configuration and certificate issuance. Istiod handles service discovery, certificate rotation for mTLS, configuration distribution to proxies, and manages both Envoy sidecars and ztunnel proxies in ambient mode.'
```

Would probably drop this last sentence since Istio doesn't integrate with Pepr (more the reverse that our operator automates integrations with Istio).